web/src/main/react-dashboard/src/components/TestRunView.js

```diff
@@ -362,6 +364,16 @@ class TestRunView extends Component {
                   )}
           ><i className="fa fa-download" aria-hidden="true"> </i> Download Test-Run log</Button>
 
+          <Button id ="tdd" size="sm" style={{marginLeft: "10px"}}
```

Comment:
Can't we replace existing test-run.log now? This blueocean log file makes the other one obsolete isnt it?

Reply:
This change will store the blueocean urls of all the builds in the future, but for older build IMO we need to keep the existing option for a while. WDYT?

Reply:
+1. It is confusing to have two links for almost the same thing. How can we fix that?